draft.xml

```diff
@@ -70,27 +49,35 @@
       </address>
     </author>
 
-    <author initials="K." surname="Duleba" fullname="Krzysztof Duleba">
+    <author fullname="Krzysztof Duleba" initials="K." surname="Duleba">
       <organization>Google Switzerland GmbH</organization>
+
       <address>
         <postal>
           <street>Brandschenkestrasse 110</street>
+
           <code>8002</code>
-          <city>Zürich</city>
+
+          <city>Z?rich</city>
```

Comment:
Revert to previous version.

Reply:
So, *currently* this won't actually work properly / render properly. 
See https://tools.ietf.org/html/rfc7997#section-3.1

The currently published draft rendered as: https://tools.ietf.org/html/draft-google-self-published-geofeeds-02#page-22

Clearly, what I have is incorrect, and I'm happy to change it to what we currently have published but we cannot get Zürich to render properly...

Reply:
I reverted the XML file to UTF-8 (in a5ec5efadd76c31a5a6b5e603b0f28289a5d0830) , and fixed this (and all of the other city / country names issues).

By the time this is published, the new RFC Format should be standard (where the XML is authoritative), and so this should be better rendable.